doc/source/installation.rst

```diff
@@ -94,6 +94,32 @@ Fedora/Rawhide:
 
      $ sudo dnf install kiwi-cli
 
+
+
+Installation for SUSE Linux Enterprise Server
```

Comment:
`SUSE Linux Enterprise Server` -> `SUSE Linux Enterprise`

(this applies to the whole SLE family)

Reply:
Fixed in commit 31b599c71.